### ISSUE TITLE
[Issue] Default colors are inaccessible on light backgrounds

### ISSUE THREAD

**Issue by pamelafox** [Azure/azure-dev#1007]
**Output from `azd version`**

azd version 0.3.0-beta.4 (commit 2f4c6d14f8a9afcdaaff769f3fa010040f1e9120)

**Describe the bug**

The CLI defaults to using colors that all fail the a11y contrast ratio on white backgrounds. The built-in Terminal on Mac defaults to a white terminal, so many developers will have that. It's also a personal preference for many.

### Example:

<img width="610" alt="Screen Shot 2022-10-26 at 5 31 18 AM" src="https://user-images.githubusercontent.com/297042/198027900-16efc5d6-60aa-480c-bf10-fd44a1882991.png">

### A11y failures:

<img width="772" alt="Screen Shot 2022-10-26 at 5 32 40 AM" src="https://user-images.githubusercontent.com/297042/198027964-50d2ec89-0ef8-4896-aeae-ed53ed3f134f.png">

<img width="758" alt="Screen Shot 2022-10-26 at 5 33 51 AM" src="https://user-images.githubusercontent.com/297042/198027956-1d6b416a-32db-4f93-8b15-75df6ff099d1.png">

<img width="751" alt="Screen Shot 2022-10-26 at 5 33 24 AM" src="https://user-images.githubusercontent.com/297042/198027959-0ab95d5e-a0c3-44ef-9b4a-be84d0befd56.png">

**Environment**

Information on your environment: Mac OSX, Monterey, Terminal

**Additional Context**

I see there's an issue in this repo about supporting NO_COLOR but that is never something I've needed to do personally, and I don't know how many developers know about it (I didn't). I think it'd be better if colors were accessible for everyone by default, if possible. 

The azure CLI also defaults to inaccessible colors on light backgrounds, which can be changed via a theme config command. It took me a lot of digging to find that command, so I don't think that approach is great either. (Relevant azure CLI issue: https://github.com/Azure/azure-cli/issues/18298)

The HiCyan is the worst offender, so perhaps at least that could be changed to a color that works on both light/dark.

**Comment by savannahostrowski** [comment]
@Austinauth Any thoughts here?

**Comment by pamelafox** [comment]
The colors come from this code:

https://github.com/Azure/azure-dev/blob/905656dea2f8c211e5e472a04525b64000e653eb/cli/azd/pkg/output/colors.go#L8

which uses a package which uses ANSI escape codes:

https://en.wikipedia.org/wiki/ANSI_escape_code#Colors

So HiCyan is "Bright Cyan" which is "20, 240, 240" in Terminal. That one has the worst contrast ratio, so if you can find another one that has decent contrast on both white/black, that would go along way.

**Comment by pamelafox** [comment]
And yeah, good point that the azure-dev code doesn't have any hardcoded hex colors -- I didn't realize that was how it worked until your comment. So we could look for an ANSI escape code row which has a color where even its extreme color values have a decent contrast ratio.

**Comment by vhvb1989** [comment]
We can set a default color palette depending on the OS (Win/Lin/Mac). Then, we can use the `config` command that @wbreza is working on to let customers override and set whatever color (HEX value maybe) they want.

IIRC, we support the env var `NO_COLOR`.  Setting this env var should remove all coloring stuff from azd

**Comment by Austinauth** [comment]
@vhvb1989 @savannahostrowski let's circle back on this today during standup. I would assume using the OS defaults would be accessible, but if not, I can provide us with some color values that will be accessible on white.

**Comment by rajeshkamal5050** [comment]
Closing in favor of #348
